src/homework/fifth/Animal.java

```diff
@@ -0,0 +1,60 @@
+package homework.fifth;
+
+public class Animal {
+    protected int eyes;
+
+    public Animal() {
+        System.out.println("I am animal");
+    }
+
+    public void eat() {
+        System.out.println("Animal eats");
+    }
+
+    public void drink() {
+        System.out.println("Animal drinks");
+    }
+}
+
+class Pet extends Animal {
+    public String name;
+    protected int tail = 1;
+    protected int raw = 4;
+
+    public Pet() {
+        System.out.println("I am pet");
```

Comment:
also we can write here 
super() 
to emphasize how constructors are called in case inheritance